airflow/helm/airflow/values.yaml

```diff
@@ -54,6 +54,7 @@ exporter:
     additionalLabels: {}
 
   statsd:
+    relayAddress: "datadog.datadog:8125"
```

Comment:
Shouldn't this be optional and only set if datadog is installed?

Reply:
udp is fire and forget so it should be fine to put it in